x/bank/keeper/send.go

```diff
@@ -67,12 +75,16 @@ func (k BaseSendKeeper) GetParams(ctx sdk.Context) (params types.Params) {
 
 // SetParams sets the total set of bank parameters.
 func (k BaseSendKeeper) SetParams(ctx sdk.Context, params types.Params) {
-	k.paramSpace.SetParamSet(ctx, &params)
+	if len(params.SendEnabled) > 0 {
+		k.SetAllSendEnabled(ctx, params.SendEnabled)
+	}
+	p := types.NewParams(params.DefaultSendEnabled)
```

Comment:
Why are we modifying default/legacy params?

Reply:
The `Params.DefaultSendEnabled` field is not deprecated.